docker-compose.arm64.yml

```diff
@@ -3,7 +3,7 @@ services:
     gateway:
         ports:
             - 8080:8080
-        image: openfaas/gateway:0.18.11-arm64
+        image: openfaas/gateway:0.18.13-arm64
```

Comment:
no 0.18.13 exists for arm64.

Reply:
That's incorrect then and shouldn't be changed

Reply:
Updated